### PR TITLE
fix e2e: add existence check

### PR DESCRIPTION
**Pull request by skylarbarrera**
## What changed (plus any additional context for devs)
fixes e2e 😅 

thought this https://github.com/rainbow-me/rainbow/pull/4310 has passed e2e other than slack previously I was mistaken 

this is sad but follows the pattern here until we can refactor this 🧶 
https://github.com/rainbow-me/rainbow/blob/0ae3218255baef535b1e15fb218614b9a75f1477/src/screens/ExplainSheet.js#L868

## What to test

e2e should pass

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Review by BrodyHughes — APPROVED** [review]
😅

File: src/screens/ExplainSheet.js [hunk 1]
```diff
@@ -536,8 +536,12 @@ export const explainers = (params, colors) => ({
   swapResetInputs: {
     button: {
       label: `Continue with ${networkInfo[params?.network]?.name}`,
-      bgColor: colors?.alpha(colors?.networkColors[params?.network], 0.06),
-      textColor: colors?.networkColors[params?.network],
+      bgColor:
+        colors?.networkColors[params?.network] &&
+        colors?.alpha(colors?.networkColors[params?.network], 0.06),
+      textColor:
```
Comment (jkadamczyk) [inline src/screens/ExplainSheet.js:542]: I have a feeling the second color doesn't need the checks. Seems like the failing part was again the chroma js

**Review by jkadamczyk — APPROVED** [review]
Just that one comment, nothing super important though you can leave it as it is just as well

File: src/screens/ExplainSheet.js (hunk 1 above)
Comment (skylarbarrera) [reply src/screens/ExplainSheet.js:542]: yeah im gonna play with adding some safeties to that fn later

**Pull request merged**